<launch>

<arg name="evaluation_service"/>

<include file="$(find abb_surrogate)/launch/evaluators/bandit_base.launch" pass_all_args="true">
    <arg name="enable_evaluation" value="true"/>
</include>

</launch>